src/lib/apiTypesV2/Entity.cpp

```diff
@@ -103,9 +103,23 @@ std::string Entity::check(ConnectionInfo* ciP, RequestType requestType)
     return "No Entity ID";
   }
 
-  if (forbiddenChars(id.c_str()))         { return "Invalid characters in entity id";        }
-  if (forbiddenChars(type.c_str()))       { return "Invalid characters in entity type";      }
-  if (forbiddenChars(isPattern.c_str()))  { return "Invalid characters in entity isPattern"; }
+  if (forbiddenChars(id.c_str()))
```

Comment:
Appart from changing from LM_E to LM_W, I wonder if with avoid "double logging", i.e. to avoid things like this:

``` 
time=2015-11-30T12:30:40.273CET | lvl=WARNING| trans=1448882505-099-00000002154 | function=forbiddenChars | comp=Orion | msg=forbiddenChars.cpp[63]: Bad Input (character '>')
time=2015-11-30T12:30:40.273CET | lvl=WARNING| trans=1448882505-099-00000002154 | function=treat | comp=Orion | msg=xmlParse.cpp[131]: Found a forbidden value in '
      tribute>'
```

which should be just:

```
time=2015-11-30T12:30:40.273CET | lvl=WARNING| trans=1448882505-099-00000002154 | function=forbiddenChars | comp=Orion | msg=forbiddenChars.cpp[63]: Bad Input (character '>')
```